cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -99,6 +105,19 @@ func SetDefaults_MasterConfiguration(obj *MasterConfiguration) {
 		obj.CertificatesDir = DefaultCertificatesDir
 	}
 
+	if obj.AuditPolicyDir == "" {
+		obj.AuditPolicyDir = DefaultAuditPolicyDir
+	}
+
+	if obj.AuditPolicyLogDir == "" {
+		obj.AuditPolicyLogDir = DefaultAuditPolicyLogDir
+	}
+
+	// TODO(chuckha) is 0 a meaningful value here? Does it mean 0 days or forever or can it be used as the empty value?
```

Comment:
Need to rationalize this comment.

Reply:
https://github.com/natefinch/lumberjack/blob/v2.0/lumberjack.go#L78

0 is a valid number and, in addition to MaxBackups == 0, means nothing will get deleted. I'll need to update this to accept 0 as valid and meaningful.